src/openzaak/utils/filterset.py

```diff
@@ -15,3 +21,67 @@ def filter_for_field(cls, field, field_name, lookup_expr=None):
         if not filter.extra.get("help_text"):
             filter.extra["help_text"] = getattr(field, "help_text", None)
         return filter
+
+
+# Updated FkOrUrlFieldFilter for nested model field filtering
+class NestedFkOrUrlFieldFilter(FkOrUrlFieldFilter):
```

Comment:
This seems like a lot of copied code from `FkOrUrlFieldFilter`, is this necessary? If needed, we could also modify the `FkOrUrlFieldFilter` from the library to support nested fields